filters_test.go

```diff
@@ -346,6 +348,68 @@ func TestMatch(t *testing.T) {
 	}
 }
 
+func TestReadInput(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		Description string
+		Input       string
+		Default     string
+		Want        string
+		ShouldFail  bool
```

Comment:
How about `wantErr`, for example, to match other tests?

Reply:
Agreed.